go_bot/modules/utils/chat_status/chat_status.go

```diff
@@ -51,20 +58,23 @@ func IsUserBanProtected(chat *ext.Chat, userId int, member *ext.ChatMember) bool
 	}
 }
 
-func IsUserAdmin(chat *ext.Chat, userId int, member *ext.ChatMember) bool {
-	if chat.Type == "private" || contains(go_bot.BotConfig.SudoUsers, strconv.Itoa(userId)) {
+func IsUserAdmin(chat *ext.Chat, userId int) bool {
+	if chat.Type == "private" {
 		return true
 	}
-	if member == nil {
-		mem, err := chat.GetMember(userId)
-		error_handling.HandleErr(err)
-		member = mem
+
+	admins, err := caching.CACHE.Get(fmt.Sprintf("admin_%v", chat.Id))
+	if err != nil {
+		admincache(chat)
 	}
-	if member.Status == "administrator" || member.Status == "creator" {
+
+	var x Cache
```

Comment:
I'd love it if you could change these var names from xyz to the first letter of the type they represent :)

Reply:
Sure!